hermeznetwork/hermez-node
# bump gin version

Closes https://github.com/hermeznetwork/hermez-node/issues/1043

### What does this PR does?

Bumps gin version


### How to test?
Run node and check, that API requests are working or run **go test api**
![Screen Shot 2021-07-28 at 6 46 20 PM](https://user-images.githubusercontent.com/5672938/127353821-4e3dee32-0316-4898-bb21-c26a2ab3de3f.png)



## Checklist

These are the criteria that every PR should meet, please check them off as you
review them:
